imagery/i.biomass/testsuite/test_i_biomass.py

```diff
@@ -0,0 +1,307 @@
+import numpy as np
+from grass.script import array
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class TestIBiomass(TestCase):
+    """Regression tests for the i.biomass GRASS GIS module."""
+
+    input_rasters = {
+        "fpar": "test_fpar",
+        "lightuse_eff": "test_lightuse_eff",
+        "latitude": "test_latitude",
+        "dayofyear": "test_dayofyear",
+        "transmissivity": "test_transmissivity",
+        "water": "test_water_availability",
+    }
+    output_raster = "biomass_output"
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up input rasters and configure test environment."""
+        cls.use_temp_region()
+        cls.runModule("g.region", n=10, s=0, e=10, w=0, rows=10, cols=10)
+        cls._create_input_rasters()
+
+    @classmethod
+    def _create_input_rasters(cls):
+        """Helper method to create all input rasters with defined expressions."""
+        expressions = {
```

Comment:
This could be perhaps the method's parameter and then you can use this method in `test_biomass_ecological_range` and maybe other places.